src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -285,6 +286,13 @@ const formConfig = {
           uiSchema: finalIncident.uiSchema,
           schema: finalIncident.schema,
         },
+        additionalRemarks781: {
+          title: 'Additional Remarks - 781',
+          path: 'new-disabilities/additional-remarks-781',
+          depends: isAnswering781Questions(0),
```

Comment:
Should this zero be hard coded? It looks like an index variable is supplied in other instances of isAnswering781Questions being used.

Reply:
We'll add a new util function for this